Application/src/controller/network/Net.java

```diff
@@ -27,7 +27,7 @@ public void run() {
     }
 
     /**
-     * Only allow a single network thread and it must be run on an FXApplicationThread.
+     * Only allow a single controller.network thread and it must be run on an FXApplicationThread.
```

Comment:
Auto replaced?

Reply:
Yes ill change that back. IntelliJ did not recognize the moving around of stuff if i did not rebuild which apparently had such side effects.